### ISSUE TITLE
Redout: GPU hang upon loading

### ISSUE THREAD

**Issue by jerbear64** [doitsujin/dxvk#237]
When trying to load Redout, I experience a GPU hang. The initial loading screen successfully works, but after the epilepsy warning `Press any key to continue` screen, the game hangs and I'm forced to reboot. 

The attached apitrace was taken with wine-pba 3.3.

### Software information
Redout: Enhanced Edition from Steam, all DLC installed. 64-bit DXVK is installed in the wineprefix.

Graphical settings were turned to the lowest before installing DXVK in the prefix.

### System information
- GPU: RX 580
- Driver: Mesa 18.0.0, LLVM 5.0.1
- Wine version: 3.5-staging, with Gallium Nine patches
- DXVK version: 0b2e88b

### Apitrace file(s)
- https://drive.google.com/open?id=10tk323chIjXOaZ-T20Xr8NFlS5nzOD2r

### Log files
- d3d11.log: https://ptpb.pw/zZ9d
- dxgi.log: https://ptpb.pw/epYZ

**Comment by jerbear64** [comment]
I managed to get an apitrace that goes to the menus. This apitrace was taken with wined3d on wine-pba 3.3, as this is the version of Wine that I've been playing the game with. The game does function on newer versions of Wine, however.

I still can't reproduce the GPU hang with it, though. I know DXVK was loaded when I replayed, as there's log files for d3dretrace and I can see the square-like artifacts on the menus that radv likes to produce.

I also tried the game with amdvlk. The game runs, but everything is black. I know amdvlk is unsupported but I figured I'd try it.

I've updated the link to the apitrace in the original post.
